SparkPost/2web2ui
# AC-1084: Link to show analytics

### What Changed
 - Added link to analytics on the dashboard list

### How To Test
 - Go to `/dashboard` on `appteam` account on staging or new account with ui feature flag `messaging_onboarding`
 - Find Getting started panel
 - Click on `Messaging` > `Show Me Sparkpost` > `Explore Analytics`
 - Check that check now appears on dashboard after navigating away

### To Do
- [ ] Double check pendo condition for popping up
